fluss-server/src/main/java/com/alibaba/fluss/server/coordinator/CoordinatorServer.java

```diff
@@ -216,7 +216,7 @@ protected CompletableFuture<Result> closeAsync(Result result) {
 
     private void registerCoordinatorLeader() throws Exception {
         // set server id
-        String serverId = UUID.randomUUID().toString();
+        String serverId = this.serverId;
         CoordinatorAddress coordinatorAddress =
                 new CoordinatorAddress(serverId, rpcServer.getHostname(), rpcServer.getPort());
```

Comment:
nit: 
```suggestion
                new CoordinatorAddress(this.serverId, rpcServer.getHostname(), rpcServer.getPort());
```
Then the comment `set server id` can be removed..